# Thruster output
# ThrusterEnabled ... { esc: bool, servo: bool }

ThrusterEnabled enabled
float64         duty_cycle  # (-1.0 ~ 1.0)
float64         angle       # [rad]
